packages/eslint-config-vue/__snapshots__/index.test.js.snap

```diff
@@ -2097,7 +2097,6 @@ Object {
     "vue/attribute-hyphenation": "error",
     "vue/attributes-order": "warn",
     "vue/comment-directive": "error",
-    "vue/component-name-in-template-casing": "warn",
```

Comment:
Updating vue's recommended config to 5.1 removes this rule.
More info on it here... https://vuejs.github.io/eslint-plugin-vue/rules/component-name-in-template-casing.html

If is still available but has been removed from 'recommended' to 'uncategorised'.

Reply:
fyi reasons for change: https://github.com/vuejs/eslint-plugin-vue/pull/703

Reply:
Ah nice one, thanks.